### ISSUE TITLE
Where is your regularization part code?

### ISSUE THREAD

**Issue by Billy1900** [shengliu66/ELR#8]
I am a little confused about the code and I want to find out the regularization part. I thought it should have existed in your loss function, but it seems like nothing there.

And the code in `train.py` might be the thing I want?
```
    trainable_params = filter(lambda p: p.requires_grad, model.parameters())

    optimizer = config.initialize('optimizer', torch.optim, [{'params': trainable_params}])
```

could you please tell me which part should I read?

**Comment by shengliu66** [comment]
Hi, thanks for asking, I have updated the basic section of the readme file, where I update the location of the implemented loss function.

**Comment by Billy1900** [comment]
that makes sense.
